packages/web3-eth-ens/src/resolver.ts

```diff
@@ -29,13 +29,48 @@ import { namehash } from './utils.js';
 //  Default public resolver
 //  https://github.com/ensdomains/resolvers/blob/master/contracts/PublicResolver.sol
 
+
+import { LRUCache } from './LRUCache.js';
+
+// Define the cache capacity
+const CACHE_CAPACITY = 100; // You can adjust this number as needed
+const resolverCache = new LRUCache<string, any>(CACHE_CAPACITY);
```

Comment:
pls make capacity configurable by users by adding resolverCache in Resolver class and add getter & setter for capacity if any one want to use it. The default should be 100.